tests/Unit/ApparentHorizons/Test_Tags.cpp

```diff
@@ -159,7 +163,9 @@ void test_radius_and_derivs() {
   }
   const auto& strahlkorper_laplacian =
       db::get<StrahlkorperTags::LaplacianRadius<Frame::Inertial>>(box);
-  CHECK_ITERABLE_APPROX(strahlkorper_laplacian, expected_laplacian);
+  Approx numerical_approx = Approx::custom().epsilon(1.0e-10).scale(1.);
+  CHECK_ITERABLE_CUSTOM_APPROX(strahlkorper_laplacian, expected_laplacian,
```

Comment:
What prompted this change?  I don't think anything should have affected the calculation of the Laplacian.

Reply:
That was changed to truncate some of the `spin_vector` results that I was trying to debug in my test and I forgot to undo the change. I'll change that back with the new updates.